features/regexp-escape.yml

```
@@ -0,0 +1,5 @@
+name: RegExp.escape
+description: The `RegExp.escape()` method replaces characters that are potentially special characters in a regular expression with equivalent escape sequences.
```

I'm not sure I understand what this does from the description. Do I have this right?

```suggestion
description: The `RegExp.escape()` static method takes a string and replaces any characters that are potentially special characters of a regular expression with equivalent escape sequences. For example, `RegExp.escape("[abc]")` returns `"\\[abc\\]"`.
```

---

Yep, that's exactly it.